### ISSUE TITLE
enforce check is failed when value is multi fields

### ISSUE THREAD

**Issue by zenglhua** [thias/puppet-sysctl#57]
For examples:

net.ipv4.tcp_wmem=4096 65536 16777216
net.ipv4.tcp_rmem=4096 87380 16777216

**Comment by tpdownes** [comment]
@zenglhua I resolved this in

https://forge.puppet.com/tpdownes/sysctl

You just have to follow the docs and use single spaces between the fields.

**Comment by gorantornqvist** [comment]
But that module doesnt work with Puppet 3.8 right?

**Comment by tpdownes** [comment]
No. Deliberately not so. You might choose it for inspiration. But... I've decided to adopt

https://forge.puppet.com/herculesteam/augeasproviders_sysctl

which claims Puppet 3.x compatibility. Then I build a really simple profile around it. You'd need to remove the Puppet 4 parameter type enforcement. I've found it to be basically drop-in.
```
class profile::sysctl(
  Hash $values = undef) {

  create_resources('sysctl', $values)

}
```
